### DISCUSSION TITLE
better playlist edit

### DISCUSSION THREAD

**Discussion by fess932** [navidrome/navidrome#2415] [Q&A] [answered]
how add all albums from artist, or how add multiple (by checkbox) albums to playlist? it's annoying to add hundreds of albums to the playlist, each requires its own operation

**Comment by metalheim** [comment, marked as answer]
Maybe a smart playlist would be better in this case.
You could f.e. create a smart playlist (see #1417 for details) - the following would create a playlist with all songs from artist "The Beatles":

```
{
  "all": [
    {"is": {"artist": "The Beatles"}}
  ],
  "sort": "album",
  "order": "desc"
}
```
